src/main/kotlin/com/launchdarkly/intellij/hover/HoverDocumentationProvider.kt

```diff
@@ -20,7 +20,7 @@ import java.io.StringWriter
 
 class HoverDocumentationProvider : AbstractDocumentationProvider() {
     private fun getFlagConfig(contextElement: PsiElement, flagKey: String): FlagConfiguration? {
-        return contextElement?.project?.service<FlagStore>()?.flagConfigs[flagKey]
+        return contextElement.project.service<FlagStore>().flagConfigs[flagKey]
```

Comment:
Using bracket notation on nullable object became an error in recent Kotlin. Could've replaced with `.get(flagKey)`, but `contextElement` is guaranteed to be non-null, so entire null-safe chain in fact had no reason.